manifests/metallb-frr.yaml

```diff
@@ -538,6 +526,8 @@ spec:
           volumeMounts:
             - name: reloader
               mountPath: /etc/frr_reloader
+            - name: frr-conf
+              mountPath: /etc/frr
```

Comment:
why is this needed?

Reply:
probably a leftover of me hammering through making it work.
Will remove it.

Reply:
Removed